src/main/java/rwg/config/ConfigRWG.java

```diff
@@ -59,6 +60,7 @@ public static void init(FMLPreInitializationEvent event) {
             generateVillages = config.getBoolean("Generate Villages", "Settings", true, "");
             generateUndergroundLakes = config.getBoolean("Generate Underground Lakes", "Settings", true, "");
             generateUndergroundLavaLakes = config.getBoolean("Generate Underground Lava Lakes", "Settings", true, "");
+            suptcbig = config.getBoolean("Support TC Biome big", "Settings", true, "");
```

Comment:
And more readable configuration labels are also welcome. 
```suggestion
            generateLargeThaumcraftBiomes = config.getBoolean("Generate large Thaumcraft biomes", "Settings", true, "");
```

P. S. *Feel free to ask for help* if you struggle with naming 😸